content/03.methods.md

```diff
@@ -21,30 +22,108 @@ We examined performance on the highlighted subgraph; however, the long-term visi
-Within these four categories each pair is randomly assigned their own individual partition rank (a continuous number between 0 and 1).
-Any rank lower than 0.7 is sorted into the training set, while any rank greater than 0.7 and lower than 0.9 is assigned to the tuning set.
-The rest of the pairs with a rank greater than or equal to 0.9 is assigned to the test set.
-Sentences that contain more than one co-mention pair are treated as multiple individual candidates.
-We hand labeled five hundred to a thousand candidate sentences of each edge type to obtain a ground truth set (Supplemental Table {@tbl:candidate-sentences})[^1].
-
-[^1]: Labeled sentences are available [here](https://github.com/greenelab/text_mined_hetnet_manuscript/tree/master/supplementary_materials/annotated_sentences).
+We used PubTator Central [@doi:10.1093/nar/gkz389] as input to our analysis.
+PubTator Central provides MEDLINE abstracts that have been annotated with well-established entity recognition tools including Tagger One [@doi:10.1093/bioinformatics/btw343] for disease, chemical and cell line entities, tmVar [@doi:10.1093/bioinformatics/btx541] for genetic variation tagging, GNormPlus [@doi:10.1155/2015/918710] for gene entities and SR4GN [@doi:10.1371/journal.pone.0038460] for species entities.
+We downloaded PubTator Central on March 1, 2020, at which point it contained approximately 30,000,000 documents.
+After downloading, we filtered out annotated entities that were not contained in Hetionet v1.
+We extracted sentences with two or more annotations and termed these sentences as candidate sentences.
+We used the Spacy's English natural language processing (NLP) pipeline (en\_core\_web\_sm) [@spacy2] to generate dependency trees and parts of speech tags for every extracted candidate sentence.
+Each candidate sentence was stratified by their corresponding abstract ID to produce a training set, tuning set, and a testing set.
+We used random assortment to assign dataset labels to each abstract. 
+Every abstract had a 70% chance of being labeled training, 20% chance of being labeled tuning, and 10% chance of being labeled testing.
+Despite the power of data programming, all text mining systems need to have ground truth labels to be well-calibrated.
+We hand-labeled five hundred to a thousand candidate sentences of each edge type to obtain a ground truth set (Table {@tbl:candidate-sentences}).
+
+| Relationship | Train | Tune | Test |
+| :--- | :---: | :---: | :---: |
+| Disease Associates Gene | 2.49 M | 696K (397+, 603-) | 348K (351+, 649-) |
+| Compound Binds Gene | 2.4M | 684K (37+, 463-) | 341k (31+, 469-) |
+| Compound Treats Disease | 1.5M | 441K (96+, 404-) | 223K (112+, 388-) |
+| Gene Interacts Gene | 11.2M | 2.19M (60+, 440-) | 1.62M (76+, 424-) |
+
+Table: Statistics of Candidate Sentences. 
+We sorted each candidate sentence into a training, tuning and testing set.
```

Comment:
Just to check, I think this contradicts line 33 above about assignment happening at the abstract level. Does it need to be updated?